src/repo/stat.js

```diff
@@ -41,5 +41,21 @@ module.exports = (createCommon, options) => {
         expectIsRepo(null, res)
       })
     })
+
+    it('should get human readable repo stats', async () => {
+      const res = await ipfs.repo.stat({ human: true })
+
+      expect(res).to.exist()
+      expect(res).to.have.a.property('numObjects')
+      expect(res).to.have.a.property('repoSize')
+      expect(res).to.have.a.property('repoPath')
+      expect(res).to.have.a.property('version')
+      expect(res).to.have.a.property('storageMax')
+      expect(res.numObjects).to.be.a('number')
+      expect(res.repoSize).to.match(/\s.?B$/gm)
+      expect(res.storageMax).to.match(/\s.?B$/gm)
```

Comment:
`/\s.?B$/gm` - hmm that doesn't look right to me.

<img width="368" alt="Screenshot 2019-11-18 at 15 32 19" src="https://user-images.githubusercontent.com/152863/69066075-a3b0fe80-0a18-11ea-86b9-87d6c8b9ee8d.png">

Are the `gm` modifiers needed? Does the test pass? If not, please ensure the test passes locally before reuesting a review. If you need any help linking and running only this test then ping me on IRC.

Reply:
Yap, sorry, the modifiers are not needed. Furthermore, I'll update the expression, I'll make it more specific.
I wouldn't have asked for a second review if the test was not passing locally. You can see it passing here:
![image](https://user-images.githubusercontent.com/11461281/69138271-43709a00-0ab6-11ea-86f5-220011da2d5b.png)


This is the `res` object

```js
{
   ...
   repoSize: '30.1 KB'
   storageMax: '9.01 PB'
}
```

My regex was matching: 
![image](https://user-images.githubusercontent.com/11461281/69139709-fe9a3280-0ab8-11ea-9a3e-fc157662c758.png)
and
![image](https://user-images.githubusercontent.com/11461281/69138196-21771780-0ab6-11ea-9ca3-7c118de932f7.png)

What do you think about this regex? Does this one seem better? It would match the whole string instead of just the units.
`/^[\d]+(\.[\d]+)?\s[KMGTP]?B$/`

Considering the `res` object (mentioned above), this new regex will match:
![image](https://user-images.githubusercontent.com/11461281/69140335-39509a80-0aba-11ea-99de-26d220761354.png)
and
![image](https://user-images.githubusercontent.com/11461281/69140457-7321a100-0aba-11ea-83d7-dc0827d9f949.png)